docs/REST API/rest-api.md

```diff
@@ -2995,6 +2999,48 @@ Requires [Fleetd](https://fleetdm.com/docs/using-fleet/fleetd), the osquery mana
   ]
 }
 ```
+---
+
+### Get host's software
+
+Retrieves a host's software.
+
+`GET /api/v1/fleet/hosts/:id/software`
+
+#### Parameters
+
+| Name       | Type              | In   | Description                                                                   |
+| ---------- | ----------------- | ---- | ----------------------------------------------------------------------------- |
+| id         | integer           | path | **Required**. The host's `id`.                                                |
+
+#### Example
+
+`GET /api/v1/fleet/hosts/1/software`
```

Comment:
Should we make this `/software/titles` to leave the door open for `/versions` later?

Up to you. `/software` is simpler.

Reply:
Also, what does the API response look like if the host has managed software?

Reply:
I think it makes sense to use `hosts/{id}/software/titles` to be more consistent with `/software/titles`.